eventsources/sources/pulsar/start.go

```diff
@@ -107,6 +107,17 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 		clientOpt.Authentication = pulsar.NewAuthenticationToken(token)
 	}
 
+	if pulsarEventSource.AuthAthenzParams != nil && pulsarEventSource.AuthAthenzSecret != nil {
```

Comment:
Is there a better way to tell if the `AuthAthenzParams` is correctly configured? e.g. if it contains a required field?

If there's not a better way, I suggest to check it with `len(pulsarEventSource.AuthAthenzParams) > 0`.

Reply:
Thanks for the feedback. I will gratefully accept your proposal.